mkdocs.yml

```diff
@@ -74,8 +74,8 @@ markdown_extensions:
   - pymdownx.caret
   - pymdownx.critic
   - pymdownx.details
-  - pymdownx.emoji:
-      emoji_generator: !!python/name:pymdownx.emoji.to_svg
+#  - pymdownx.emoji:
+#      emoji_generator: !!python/name:pymdownx.emoji.to_svg
```

Comment:
My mkdocs could not find this. I'm not sure if we even use it.
I'm also not sure what it does, or wether we use it.

Reply:
Pretty sure this is related to inline emoji (unicode chars) to graphic translation.